test_common/harness/testHarness.cpp

```diff
@@ -67,6 +67,21 @@ int runTestHarness( int argc, const char *argv[], int testNum, test_definition t
                           ( imageSupportRequired ) ? verifyImageSupport : NULL );
 }
 
+int skip_init_info() {
+    log_info("Test skipped while initialization\n");
+    log_info("PASSED %s.\n", "test");
```

Comment:
As per previous comments, I think my preference would be to say `SKIPPED` here with the total number of tests. We should have a clear distinction between passing and skipping tests.

Also, I don't understand why `test` is printed via a format string.

Reply:
regarding `test` - its a mistake.  Are we interested in printing full list of tests skipped ?

Reply:
if no list of tests then:

Test skipped (requires at least version 2.1, but the device reports version 1.2)
Test skipped while initialization
SKIPPED 2 of 2 tests.

Reply:
I wouldn't try to print the list. This really belongs in the JSON results. The sample output looks good to me.

Reply:
updated